fake/Cargo.toml

```diff
@@ -79,3 +77,6 @@ required-features = [
 name = "usage"
 path = "examples/usage.rs"
 required-features = ["derive"]
+
+[package.metadata.cargo-all-features]
+denylist = ["geo-types", "serde_json"]
```

Comment:
This can also be solved by adding these to `[features]` so their other co-dependencies can be specified